shell/platform/embedder/embedder.h

```diff
@@ -1161,6 +1267,21 @@ typedef struct {
   FlutterSemanticsCustomAction* custom_actions;
 } FlutterSemanticsUpdate;
 
+/// A batch of updates to semantics nodes and custom actions.
+typedef struct {
+  /// The size of the struct. Must be sizeof(FlutterSemanticsUpdate2).
+  size_t struct_size;
+  /// The number of semantics node updates.
+  size_t nodes_count;
```

Comment:
We should probably leave these as-is to make transitioning from the old one easier, but the nitpicker in me really wants to say `node_count` and `custom_action_count` :)

Reply:
I updated `FlutterSemanticsUpdate2` but let `FlutterSemanticsUpdate` as-is.